plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerConnectorTest.java

```diff
@@ -345,6 +349,59 @@ public void testPredicatePushdown()
         }
```

Comment:
enable `SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_EQUALITY`

Reply:
I think can not, because this feature enabled only with toggle